nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java

```diff
@@ -857,19 +853,7 @@ private void openParFile(File parDirFile) throws IOException {
                     BrukerPar.processBrukerParFile(parMap, acquFile.toString(), i + 1, false);
                     Integer iPar;
                     if ((iPar = getParInt("TD," + (i + 1))) != null) {
-                        if (iPar > 1) {
-                            acqdim++;
-                        } else {
-                            if ((iPar = getParInt("NusTD," + (i + 1))) != null) {
-                                if (iPar > 1) {
-                                    acqdim++;
-                                } else {
-                                    acqdim++;
-                                }
-                            } else {
-                                acqdim++;
-                            }
-                        }
+                        acqdim++;
```

Comment:
Removing the identical branch resulted in another identical branch which resulted in another identical branch, so removed all three since acqdim always increased